cli/dcoscli/node/main.py

```diff
@@ -748,20 +750,25 @@ def _ssh(leader, slave, option, config_file, user, master_proxy, command):
     if command is None:
         command = ''
 
-    master_public_ip = dcos_client.metadata().get('PUBLIC_IPV4')
-    if master_proxy:
+    proxy_ip = proxy_ip or config.get_config_val('core.dcos_proxy_ip')
```

Comment:
Not a huge fan of adding another config param because it is only used for one command, which already has a flag.